app/components/primer/button_component.rb

```diff
@@ -20,7 +20,7 @@ class ButtonComponent < Primer::Component
     VARIANT_OPTIONS = VARIANT_MAPPINGS.keys
 
     DEFAULT_TAG = :button
-    TAG_OPTIONS = [DEFAULT_TAG, :a].freeze
+    TAG_OPTIONS = [DEFAULT_TAG, :a, :summary].freeze
```

Comment:
Ha, just realizing we don't support `<input type="button">`. Haven't used that for years anyway!

Reply:
wow, that's true!
when I created this component I tried to make it as restrict as possible, only allowing tags I needed at the moment
I didn't need the `input` tag so it got ignored 🤷 

We should add support for it on a separate PR

Reply:
> We should add support for it

We should?? I was thinking the use of `<button>` was preferred over `<input type="button">`.

Reply:
now I'm confused hahaha
maybe you are right, `button` + `role` should work fine 🤔